syntax.d/coffeescript.ini

```diff
@@ -0,0 +1,9 @@
+name=CoffeeScript
+extensions=coffee, litcoffee
+highlight_numbers=true
+singleline_comment_start=#
+multiline_comment_delims=###, ###
+singleline_string_quotes="
+multiline_string_delim="
+keywords_1=this, new, if, else, return, try, catch, throw, function, class, extends, super, switch, case, default, break, continue, for, in, while, do, unless, until
+keywords_2=true, false, null, undefined
```

Comment:
Is there a reference page that lists all keywords for CoffeeScript (and the other languages as well)? If so, can you add it as a comment, like in https://github.com/ilai-deutel/kibi/blob/master/syntax.d/python.ini#L7?

Reply:
I'm not aware of any single reference page that listed all keywords for CoffeeScript.